index.html

```diff
@@ -203,6 +203,41 @@ <h1 class="introduction">I’m Mariane Glad to see you!</h1>
       <li class="topic item">Skills</li>
     </ul>
   </div>
+
+  <footer class="footer">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it
+      </p>
+    </div>
+
+    <div class="contact-forms">
+      <form class="message-form" action="https://formspree.io/f/xqknrzed" method="post">
+
+        <div class="input-li">
```

Comment:
- [ OPTIONAL] I believe you can remove the wrapper `div` from the form elements since the `form` input elements are block level elements and achieve the same thing. But, you will have cleaner DOM if you remove it.